arc/conformance/plugin/conformance.yaml

```diff
@@ -0,0 +1,15 @@
+sonobuoy-config:
+  driver: Job
+  plugin-name: akv-secrets-provider-arc-conformance
+  result-format: junit
+spec:
+  image: mcr.microsoft.com/oss/azure/secrets-store/provider-azure-arc-conformance:v1.0.1
```

Comment:
do we need this image to be in mcr or just staging registry is enough?

Reply:
is our staging registry publically accessible?

Reply:
Yes, it is!